extensions/objects/models/objects_test.py

```diff
@@ -927,37 +927,24 @@ def test_translatable_objects_naming(self):
                     continue
                 self.assertEqual(name.index('Translatable'), 0)
             else:
-                self.assertTrue('Translatable' not in name)
-
-    def test_translatable_required_methods(self):
-        mock_translatable_object_class = type(
-            python_utils.convert_to_bytes('mock_translatable_object_class'),
-            (objects.BaseTranslatableObject,),
-            {})
+                self.assertNotIn('Translatable', name)
 
+    def test_using_required_methods_with_implementation_raises_error(self):
```

Comment:
I still don't understand this name, sorry. Do you mean something like test_abstract_base_class_raises_not_implemented_error()?

> I wanted to specifically test that a subclass of BaseTranslatableObject would throw the NotImplementedError.

If you want to do this, make a mock subclass (maybe a private one just for this test), and test calling methods on that instead.

Reply:
sorry, I meant 'test_using_required_methods_**without**_implementation_raises_error', but I will go with ```test_abstract_base_class_raises_not_implemented_error```